### DISCUSSION TITLE
How to install Git extension on PhoenixCode?

### DISCUSSION THREAD

**Discussion by SimoneGuidi** [phcode-dev/phoenix#1881] [Q&A]
Hi, I have a website created locally that I would need to transfer to a git repository, I would like to do it directly from phoneix code, so that future commits I can do them directly from the ide, but I can't find the extension. How do I do this?

**Comment by abose** [comment]
We are working on GIT support directly in Phoenix code and it should be available in the next release on October.

Currently, git is not integrated in Phoenix code.
